tests/phpunit/tests/interactivity-api/wpInteractivityAPI.php

```diff
@@ -1375,4 +1375,136 @@ public function test_kebab_to_camel_case() {
 		$this->assertSame( 'myItem', $method->invoke( $this->interactivity, '-my-item-' ) );
 	}
+
+	/**
+	 * Tests that `wp_interactivity_get_element` returns an array with the
+	 * current element's attributes.
+	 *
+	 * @ticket 62136
+	 *
+	 * @covers wp_interactivity_get_element
+	 * @covers ::process_directives
+	 */
+	public function test_get_element_returns_current_element_representation() {
+		/*
+		 * The global WP_Interactivity_API instance is momentarily replaced to
+		 * make the global function `wp_interactivity_get_element` work as expected.
+		 */
+		global $wp_interactivity;
+		$wp_interactivity_prev = $wp_interactivity;
+		$wp_interactivity      = $this->interactivity;
+
+		$this->interactivity->state(
+			'myPlugin',
+			array(
+				'dataTest' => function () {
+					$element = wp_interactivity_get_element();
+					return $element['attributes']['data-test'];
+				},
+			)
+		);
+
+		$html           = '
+			<section data-wp-interactive="myPlugin">
+				<div class="buttons">
+					<button
+						class="button"
+						data-test="button 1"
+						data-wp-bind--data-test-value="state.dataTest"
+					></button>
+					<button
+						class="button"
+						data-test="button 2"
+						data-wp-bind--data-test-value="state.dataTest"
+					></button>
+				</div>
+			</section>
+		';
+		$processed_html = $this->interactivity->process_directives( $html );
+		$p              = new WP_HTML_Tag_Processor( $processed_html );
+		$p->next_tag( 'button' );
+		$this->assertSame( 'button 1', $p->get_attribute( 'data-test-value' ) );
+		$p->next_tag( 'button' );
+		$this->assertSame( 'button 2', $p->get_attribute( 'data-test-value' ) );
+
+		// Restore the original WP_Interactivity_API instance.
+		$wp_interactivity = $wp_interactivity_prev;
+	}
+
+	/**
+	 * Tests that the attributes returned by `wp_interactivity_get_element` are
+	 * those originally present before directives are processed.
+	 *
+	 * @ticket 62136
+	 *
+	 * @covers wp_interactivity_get_element
+	 * @covers ::process_directives
+	 */
+	public function test_get_element_returns_original_attributes_only() {
+		/*
+		 * The global WP_Interactivity_API instance is momentarily replaced to
+		 * make the global function `wp_interactivity_get_element` work as expected.
+		 */
+		global $wp_interactivity;
+		$wp_interactivity_prev = $wp_interactivity;
+		$wp_interactivity      = $this->interactivity;
+
+		$attributes = null;
+
+		$this->interactivity->state(
+			'myPlugin',
+			array(
+				'processAttributes' => function () use ( &$attributes ) {
+					$element = wp_interactivity_get_element();
+					$attributes = $element['attributes'];
+					return 'processed';
+				},
+			)
+		);
+
+		$html           = '
+			<section data-wp-interactive="myPlugin">
+				<div class="buttons">
+					<button
+						disabled
+						class="original"
+						data-attr="original"
+						data-wp-bind--data-attr="state.processAttributes"
+					></button>
+				</div>
+			</section>
+		';
```

Comment:
Same here:

```suggestion
		$html           = <<<HTML
			<section data-wp-interactive="myPlugin">
				<div class="buttons">
					<button
						disabled
						class="original"
						data-attr="original"
						data-wp-bind--data-attr="state.processAttributes"
					></button>
				</div>
			</section>
HTML;
```

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/7460/commits/2aa912cef7432f4a6b9f20bf430a52775555c08a.